CODEOWNERS

```diff
@@ -1,4 +1,5 @@
 * @solarwindscloud/eng-apm-instrumentation
 
-# placeholder for GH EMU where this will move to new org for public repos
-#* @solarwinds/eng-apm-instrumentation
+# prepare for move to solarwinds
+@solarwinds/eng-pub-apm-instrumentation
```

Comment:
This won't work, right now it matches a file named `@solarwinds/eng-pub-apm-instrumentation` and doesn't assign it to everyone.
```suggestion
# prepare for move to solarwinds
* @solarwinds/eng-pub-apm-instrumentation
```

Reply:
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax